delivery.yaml

```diff
@@ -318,6 +340,230 @@ pipeline:
               cpu: 500m
               memory: 1Gi
 
+- id: create-opa-enabled-cluster
```

Comment:
this PR time

```
Total steps runtime: 1 h, 34 m, 26 s
```

other PR times (went through 5 not all all of them)

```
Total steps runtime: 1 h, 8 m, 47 s
```

Reply:
Hmm.. I understand that it would be a concern. Will wait for Teapot's feedback for that. However we do not hope to keep these steps forever. It's only until we are confident enough to enable this feature for all the clusters. 

Also the overall completion time for a build is not affected as these 3 steps run parallel to other create-cluster, e2e steps.

Reply:
> we do not hope to keep these steps forever. It's only until we are confident enough to enable this feature for all the clusters.

The e2e environment is the same for all the infrastructure, it's not dependent on channels and having something we're unsure about is fine to have as a work in progress. Additionally, we can also iterate over the e2e tests safely, so we can merge this version and add improvements later on. Just have to keep backwards compatibility in mind, so as long as the existing test suite is not "disturbed", it's fine to merge this, I'd say.

 So, I'd recommend having `SKIPPER_OPA_ENABLED: true` for the existing cluster and integrating the OPA test into the exiting tests. 

We can evaluate and see how to manage the longer build time. Two clusters add too much complexity to keep the setup sustainable.

Reply:
I already found one potential setup that will definitely make a big impact on the running time of the OPA tests, please see the comment below.